recipe/meta.yaml

```diff
@@ -35,7 +35,7 @@ source:
 
 build:
   number: 1
```

Comment:
I think I prefer to keep things as is.

Reply:
second that - this is intentional (although obv up for debate). It's to keep things aligned with conda-forge. Otherwise we get unintentional priority if a user has both us and conda-forge as channels in their config (which is often)